package.json

```diff
@@ -65,15 +65,15 @@
     "url": "https://github.com/ipfs/js-ipfs-api"
   },
   "devDependencies": {
-    "aegir": "^13.0.0",
+    "aegir": "https://github.com/ipfs/aegir#karma-webpack-bundle-alt2",
```

Comment:
Can you release aegir and update it here?

Reply:
I'll do once I'm done with my Windows fixes.

Reply:
Which ones?

Reply:
The ones I'm working on atm.